spec/arrays.dd

```diff
@@ -788,6 +788,10 @@ $(H3 $(LNAME2 void-initialization, Void Initialization))
         Void initializations are an advanced technique and should only be used
         when profiling indicates that it matters.
         )
+        $(P to void initialise the elements of dynamic array use
```

Comment:
`to` should have been capitalized. But I wasn't fast enough.

Reply:
Also, shouldn't it be elements of __a__ dynamic array?